tests/TestCase.php

```diff
@@ -17,22 +18,27 @@ abstract class TestCase extends OrchestraTestCase
 {
     use RefreshDatabase, WithWorkbench;
 
+    private Core $core;
+
     protected function setUp(): void
     {
         parent::setUp();
 
-        $core = $this->app->make(Core::class);
-        $core->state->reset();
-        $core->clock->microtimeResolver = fn () => (float) now()->format('U.u');
-
-        $this->app->make(Repository::class)->set('nightwatch.error_log_channel', 'null');
+        $this->core = $this->app->make(Core::class);
+        $this->core->state->reset();
+        $this->core->clock->microtimeResolver = fn () => (float) now()->format('U.u');
+        Nightwatch::handleUnrecoverableExceptionsUsing(fn ($e) => throw $e);
```

Comment:
Unrecoverable exceptions that occur within a test are now thrown and the test will fail. This is default behaviour. A custom handler can be registered within the test.